src/geocoding/tests/test_geocoding.py

```diff
@@ -108,8 +108,8 @@ def test_entity_add_city_shape(air_quality_observed):
 
     geo = r['location']['value']
     assert geo['type'] == 'Polygon'
-    assert len(geo['coordinates']) == 1
-    assert len(geo['coordinates'][0]) == pytest.approx(2186, abs=100)
+    polygon = geojson.Polygon(geo['coordinates'])
+    assert polygon.is_valid
 
```

Comment:
so we now just check that the polygon is indeed a polygon regardless what polygon and what polygon, right?